rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
+
+//problema 2
+function Get2LargestNo(vect)
+{
+    var numbers = [];
+    numbers[0] = -Infinity;
+    numbers[1] = -Infinity;
+
+    for(i=0; i < vect.length; i++)
+    {
+        if(numbers[1] < vect[i])
+        {
+            numbers[0] = numbers[1];
+            numbers[1] = vect[i];
+        }
+    }
+
+    return numbers;
+}
```

Comment:
Sorry to say but this implementations doesn't work. For this test data: ``[1,2,3,6,5,4]``, the result your function gives is ``[3, 6]``